libraries/render-utils/src/AntialiasingEffect.cpp

```diff
@@ -140,7 +140,7 @@ void Antialiasing::run(const render::RenderContextPointer& renderContext, const
 #else
 
 void AntialiasingConfig::setAAMode(int mode) {
-    _mode = std::min((int)AntialiasingConfig::MODE_COUNT, std::max(0, mode));
+    _mode = std::min((int)AntialiasingConfig::MODE_COUNT, std::max(0, mode)); // Just use unsigned?
```

Comment:
Ints are just more comfortable to work with, I figure. You have to take the problems of it being unsigned everywhere.

Reply:
I find unsigned ints much more intuitive for enums, but I guess it doesn't matter. This particular line just seems like a waste so I wanted to flag it for you.

Reply:
Works.  And I must say, having been involved with this engine and project for several years, Vircadia at 2x resolution on Oculus Quest (Link) in VR with antaliasing off, all effects off, looks so incredibly clean.  The moment any effects are turned out there are huge amounts of aliasing introduced.